app.js

```diff
@@ -108,11 +108,25 @@ function createTableFooter () {
   let dailyTotal = document.createElement('th');
   dailyTotal.innerText = sum2;
   tableRow2.appendChild(dailyTotal);
-  console.log(sum2);
 }
 
 createTableFooter();
 
+function inputNewStore(){
+  let form = document.getElementById('form');
+  form.addEventListener('submit',addStore);
+  function addStore(event) {
+    event.preventDefault();
+    let form = event.target;
+    let city = form.city.value;
+    let minCust = form.minCust.value;
+    let maxCust = form.maxCust.value;
+    let avgSale = form.avgSale.value;
+    let newStore = new Locations(city, minCust, maxCust, avgSale);
```

Comment:
If you wanted, this could have been abbreviated to `new Locations(form.city.value, form.minCust.value, form.maxCust.value, form.avgSale,value)`